src/fetchers/wakatime-fetcher.js

```diff
@@ -4,7 +4,7 @@ const fetchLast7Days = async ({ username, api_domain }) => {
   try {
     const { data } = await axios.get(
       `https://${
-        api_domain || "wakatime.com"
+        api_domain && api_domain.replace(/[^a-z-.0-9]/gi, "") || "wakatime.com"
```

Comment:
```suggestion
        api_domain?.replace(/[^a-z-.0-9]/gi, "") || "wakatime.com"
```

I think the code change you suggested can be organized in this way as well. : )

Or, like this

```suggestion
        api_domain ? api_domain.replace(/[^a-z-.0-9]/gi, "") : "wakatime.com"
```